tt_metal/impl/dispatch/cq_commands.hpp

```diff
@@ -162,6 +162,8 @@ struct CQDispatchWritePagedCmd {
 constexpr uint32_t CQ_DISPATCH_CMD_PACKED_WRITE_FLAG_NONE      = 0x00;
 constexpr uint32_t CQ_DISPATCH_CMD_PACKED_WRITE_FLAG_MCAST     = 0x01;
 constexpr uint32_t CQ_DISPATCH_CMD_PACKED_WRITE_FLAG_NO_STRIDE = 0x02;
+
+constexpr uint32_t CQ_DISPATCH_CMD_PACKED_WRITE_MAX_CORES = 108;  // GS 120 - 1 row TODO: this should be a compile time arg passed in from host
```

Comment:
MAX_CORES is really a side affect of how this number was deduced: wanted to handle enough cores to do a packed_write to all cores.  However, packed write doesn't have to iterate over cores, really this is the MAX_SUBCMDS.  I think here would be a good place to make 2 constexpr, one for unicast one for multicast the latter calculated from the former.